lisp/ess-custom.el

```diff
@@ -2321,41 +2325,88 @@ and `ess-tracebug'.")
 
 ;;*;; Inferior ESS commands
 
-(defvar ess-load-command "source(\"%s\")\n"
+(defvar ess-load-command "source('%s')\n"
   "Dialect specific format-string for building the ess command to load a file.
 
 This format string should use %s to substitute a file name and should
 result in an ESS expression that will command the inferior ESS to load
 that file.")
-(define-obsolete-variable-alias 'inferior-ess-load-command 'ess-load-command "ESS v13.09")
-
-(defvar ess-load-visibly-command nil
```

Comment:
> or writing a formatting function and binding it to ess-format-load/eval-command-function in the customise-alist. 

What is the point of this? Do you have a particular use case when those must be functions?

ESS always dealt with two types of evaluation - visual and non-visual. I think this is a well defined and non-controversial scheme. It might be unclean down there at constants level, but removing the distinction completely doesn't solve the problem. Quite the oposite, having only one evaluation format or function will likely to confuse things.